exec/java-exec/src/main/java/org/apache/drill/exec/planner/sql/conversion/DrillRexBuilder.java

```diff
@@ -65,9 +65,9 @@ public RexNode ensureType(
    * @return Call to CAST operator
    */
   @Override
-  public RexNode makeCast(RelDataType type, RexNode exp, boolean matchNullability) {
+  public RexNode makeCast(RelDataType type, RexNode exp, boolean matchNullability, boolean safe) {
```

Comment:
This really highlights an issue with Calcite.  They really could have added an additional function something like below and nothing would have broken...


```
makeCast(RelDataType type, RexNode exp, boolean matchNullability) {
   return makeCast(type, exp, matchNullability, false);
}
```

Reply:
🤦